code/modules/clothing/rogueclothes/neck/neck.dm

```diff
@@ -311,6 +316,28 @@
 			H.Knockdown(20)
 			H.Paralyze(20)
 
+/obj/item/clothing/neck/roguetown/psicross/silver/equipped(mob/living/user, slot, initial = FALSE, silent = FALSE)
+	. = ..()
+	if(active_item)
+		return
+	active_item = TRUE
+	if(HAS_TRAIT(user, TRAIT_ZEALOT))
+		to_chat(user, span_notice("I feel my faith in PSYDON protecting me!"))
+		ADD_TRAIT(user, TRAIT_ANTIMAGIC, TRAIT_GENERIC)
```

Comment:
I also very specifically don't think we should give them constanti anti-magic. All of our antimagic sources are time limited, be it via a temporary antimagic effect followed by a cooldown; or taking a drug that provides brief antimagicy with harmful visual effects.